package com.campusdual.classroom;

public class Exercise28 {
    public static void main(String[] args) {
        throw new RuntimeException("Esta es una excepción lanzada desde el método main");
    }
}
